setup.py

```diff
@@ -6,15 +6,15 @@
 # https://github.com/Sceptre/project/wiki/sceptre-template-handler-template
 
 # lowercase, use `-` as separator.
-TEMPLATE_HANDLER_NAME = 'custom-template-handler'
+TEMPLATE_HANDLER_NAME = 'sceptre-cdk-handler'
 # the template_handler call in sceptre e.g. type: custom_template_handler.
-TEMPLATE_HANDLER_TYPE = 'custom_template_handler'
+TEMPLATE_HANDLER_TYPE = 'cdk'
 # do not change. Rename template_handler/template_handler.py to template_handler/{TEMPLATE_HANDLER_TYPE}.py
 TEMPLATE_HANDLER_MODULE_NAME = 'template_handler.{}'.format(TEMPLATE_HANDLER_TYPE)
 # CamelCase name of template_handler class in template_handler.template_handler.
-TEMPLATE_HANDLER_CLASS = 'CustomTemplateHandler'
+TEMPLATE_HANDLER_CLASS = 'CDK'
 # One line summary description
-TEMPLATE_HANDLER_DESCRIPTION = 'A Custom Template Handler'
+TEMPLATE_HANDLER_DESCRIPTION = 'AWS CDK Template Handler'
 # if multiple use a single string with comma separated names.
 TEMPLATE_HANDLER_AUTHOR = 'Sceptre'
```

Comment:
I think you'll need the python CDK library installed, since your handler requires that. It's not just a matter of needing CDK functional in node; we also need it available to use in Python.